config/prow/cluster/build/build_kubernetes-external-secrets_customresource.yaml

```diff
@@ -3,6 +3,19 @@
 ---
 apiVersion: kubernetes-client.io/v1
 kind: ExternalSecret
+metadata:
+  name: akeyless-test-cred
+  namespace: test-pods
+spec:
+  backendType: gcpSecretsManager
+  projectId: kubernetes-upstream
```

Comment:
This `projectId` would need to reference your project. `kubernetes-upstream` is what we use for azure and it's not shared.

Reply:
done